cumulusci/cli/cci.py

```diff
@@ -268,6 +270,33 @@ def handle_exception(error, is_error_cmd, logfile_path, should_show_stacktraces=
         raise error
 
 
+def ssl_error_message(error):
+    message = "We encountered an error with SSL:" + str(error)
+    if "CERTIFICATE_VERIFY_FAILED" in str(error):
+        message += "\n\nThe CURL_CA_BUNDLE or REQUESTS_CA_BUNDLE environment variables can often be used to solve this."
+
+        if "salesforce.com" in str(error):
+            message += (
+                "\n\nIf you are a Salesforce employee attempting "
+                "to use internal resources please search Concierge for "
+                "`How to Manually Download Internal SSL Root Certificates` "
+                "and ensure that the environment variables refer to the PEM files. "
+                "You may need to concatenate them into a single file."
+            )
+
+        if "local.dev" in str(error):
+            message += (
+                "\n\nIf you are a Salesforce employee attempting "
+                "to use a local server please ensure that the key you "
+                "generated for local SSL is in the PEM files you reference."
+                "If you have trouble accessing github, pypi and other resources "
+                "you may also need to merge in the certificates from this URL: "
+                "https://raw.githubusercontent.com/certifi/python-certifi/master/certifi/cacert.pem"
```

Comment:
Seems like us providing a way to merge them from the installed release of certifi would be preferable to telling people to get them from a github repository. Have you submitted an SA to find out what approach security would like us to use?